uh-groupings-api-overrides.skeleton.properties

```diff
@@ -1,27 +1,15 @@
-# Create a directory in your home directory called ".<username for your computer>-conf".
+# Create a directory in your home directory called ".<username for your coputer>-conf".
 # Copy this file into that directory and name it uh-groupings-api-overrides.properties.
 # Fill in all blank values.
 
-# UH username
+# Provide your UH username for both of the following.
 groupings.api.localhost.user=
-# UH username
 groupings.api.test.admin_user=
 
+# Grouper client settings. Request the password via filedrop.
 grouperClient.webService.url=https://grouper-test.its.hawaii.edu/grouper-ws/servicesRest/
+grouperClient.webService.login=_groupings_api_2
+grouperClient.webService.password=
 
-# grouper app-user username
-grouperClient.webService.login =
 
-# grouper app-user password
-grouperClient.webService.password =
 
-# Override recipient email
-email.send.recipient.override=
-
-# Add the following lines below if you want to do a live reload.
-# This is not necessary in order to run the project.
-
-# Path of your uh-groupings-api project see path examples above.
-# project.base-dir=
-# spring.thymeleaf.prefix=${project.base-dir}/src/main/resources/templates/
-# spring.resources.static-locations=${project.base-dir}/src/main/resources/static/
```

Comment:
Was it discussed as a team to get rid of the live reload? It may be appreciated by future team members since it can be a bit annoying to keep rerunning the project after every change.